web-registry/src/utils/price/computeMedianPrice.ts

```diff
@@ -0,0 +1,14 @@
+type Params = {
+  prices: number[];
+};
+
+export const computeMedianPrice = ({ prices }: Params) => {
```

Comment:
are the incoming prices always ordered? maybe worth adding a comment about that in here or sort them directly in this function?

Reply:
They should already be sorted here but will double-check and add a comment. Might be safer to sort here I agree, will check that too.